server/datastore/mysql/migrations/tables/20231002120317_AddDiscardToQueries.go

```diff
@@ -0,0 +1,19 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231002120317, Down_20231002120317)
+}
+
+func Up_20231002120317(tx *sql.Tx) error {
+	_, err := tx.Exec("ALTER TABLE `queries` " +
+		"ADD COLUMN discard_data TINYINT(1) NOT NULL DEFAULT FALSE;")
```

Comment:
nit: i might be wrong, but i think this formatting follows the existing patterns:
```suggestion
	_, err := tx.Exec(`
		ALTER TABLE queries
		ADD COLUMN discard_data TINYINT(1) NOT NULL DEFAULT FALSE;
		`)
```

Reply:
also, is the default for new and existing policies going to be to save query results?

Reply:
@mostlikelee Figma designs + flow chart seem to indicate that it is the default, but I could be wrong